index.bs

```diff
@@ -7,3 +7,514 @@ URL: https://netwerk-digitaal-erfgoed.github.io/requirements-network-of-terms/
+    The consequence of this approach is that users may need to know different operators from different terminology sources.
+</div>
+
+The Network of Terms searches for terms with a specific status or parent term {#14}
+--------------------
+**Priority**: Should have
+
+Users want to find terms that best match their search query.
+This query contains one or more words, such as `Zwaard` or `Haarlem`.
+To prevent getting too many results, users want to complement their search query with additional criteria: "status" and "parent term".
+These criteria narrow the results.
+
+With "status" a user can indicate that he only wants to find terms with this status.
+For example: only approved terms, no candidate terms.
+
+With "parent term" a user can indicate that he only wants to find terms with this parent term.
+For example: only places that belong to "The Netherlands", not to "South Africa".
+
+The Network of Terms can only search for terms with a certain status or parent term if the terminology sources support this.
+If a terminology source doesn't, then the Network of Terms cannot provide this functionality for that source either.
+
+<div class="note">
+    **Discussion**: should this requirement be split into two requirements, one for "status" and one for "parent term"?
+</div>
+
+The Network of Terms ensures that terms are ordered by relevance {#15}
+--------------------
+**Priority**: Must have
+
+Users want to find terms that best match their search query.
+The Network of Terms therefore ensures that matching terms are ordered by relevance: the terms that best answer the query are returned first.
+
+The Network of Terms does not order the terms itself; the terminology sources do.
+If a terminology source does not support this functionality, the Network of Terms will return the terms in the default order of the terminology source.
+
+The Network of Terms has no knowledge of the relevance ranking of terminology sources. It therefore cannot combine the terms from sources into one relevance-ranked
+result set. Instead, it groups the terms in the result set by source, respecting the order of each.
+
+The Network of Terms returns exactly the information that the terminology sources provide, without any substantive changes {#16}
+--------------------
+**Priority**: Must have
+
+The Network of Terms allows users to search in terminology sources.
+It returns the matching terms directly to the users and does not change it.
+For example: it does not correct, enrich or re-sort terms - 'what you see is what you get'.
+
+The Network of Terms does change the data structure with which terminology sources publish their terms.
+This change is necessary in order to get a uniform, source-independent structure (see [requirement 1](#1)).
+
+The Network of Terms monitors and displays the status of terminology sources {#17}
+--------------------
+**Priority**: Must have
```

Comment:
I’m not entirely convinced of ‘must have’ here. The fact that a source is unreachable, is already clear from the results (see § 3.8). So this requirement is either:

- an internal optimization (temporarily stop querying a source if it has been down for the last period of time)
- something nice for end users.

Reply:
Good point!